### PR TITLE
GH-1625 Add Choropleth Legend

### PR DESCRIPTION
**Pull request by GordonSmith**
Add choro to multichart
Add rainbow bin count and format to rainbow legend
Tweak rainbow legend

Fixes GH-1625

Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@jbrundage Please Review:  http://rawgit.com/GordonSmith/Visualization/GH-1625/demos/dermatology.html?src/composite/MegaChart.choro&legendPosition=right&legendFormat=.1s&legendBins=12&title=US%20States%20Choropleth&chartType=CHORO_USTATES

File: src/other/Legend.js [hunk 1]
```diff
@@ -74,14 +82,7 @@
     function _htmlColorBlock(hexColor) {
         return "<div class=\"colorBlock\" style=\"background-color:" + hexColor + ";\"></div>";
     }
-    function commaSeparateNumber(val) {
```
Comment (jchambers-ln) [inline src/other/Legend.js]: is it safe to completely remove this function?

Comment (GordonSmith) [inline src/other/Legend.js]: Yes - its private to this file...

**Comment by jbrundage** [comment]
@GordonSmith Looks great

**Pull request merged**